client/html/layouts/basket/product/body-default.html

```diff
@@ -0,0 +1,114 @@
+<?php
```

Comment:
The location of the file should be "client/html/layouts/basket/mini/product-body-default.html"